Support/Testing/Patches/memory-region-info.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/packages/Python/lldbsuite/test/tools/lldb-server/gdbremote_testcase.py b/packages/Python/lldbsuite/test/tools/lldb-server/gdbremote_testcase.py
+index d63ddbe..68f4f50 100644
+--- a/packages/Python/lldbsuite/test/tools/lldb-server/gdbremote_testcase.py
++++ b/packages/Python/lldbsuite/test/tools/lldb-server/gdbremote_testcase.py
+@@ -698,7 +698,7 @@ class GdbRemoteTestCaseBase(TestBase):
+ 
+         # Validate keys are known.
+         for (key, val) in list(mem_region_dict.items()):
+-            self.assertTrue(key in ["start", "size", "permissions", "error"])
++            self.assertTrue(key in ["start", "size", "permissions", "error", "name"])
```

Comment:
Well I mean, presumably since upstream needs it, they'll want this patch. Actually how is upstream passing tests without this...